src/configs/dataset/giga_pretrain.yaml

```diff
@@ -3,7 +3,9 @@ defaults:
 
 dataset:
   _target_: tasks.contrastive_learning.contrastive_dataset.ContrastivePreTrainingDataset
-  root: ${paths.root_dir}/tasks/contrastive_learning/giga_raw_files/
+  # root: ${paths.root_dir}/tasks/contrastive_learning/giga_raw_files/
+  root: /u201/mhao/MDT-2-1/src/tasks/contrastive_learning/giga_raw_files/
```

Comment:
Can you expand why this is need from the original version? `paths.root_dir` should resolve to be `/u201/mhao/MDT-2-1/src`